changelog.d/2394.doc.rst

```diff
@@ -0,0 +1,3 @@
+Extended towncrier news template to include change note categories.
+This allows to see what types of changes a given version introdueces
```

Comment:
Minor typo: s/introdueces/introduces/

Reply:
Thanks!
P.S. You can send suggested changes in PR comments so that I could accept them from the GitHub UI.